GoProjects/CalorieCounter/food/food.go

```diff
@@ -1,7 +1,37 @@
 package food
 type Food struct {
-	Name string `json:"name"`
-	Category string `json:"category"`
-	Calories int `json:"calories"`
+	name string `json:"name"`
+	category string `json:"category"`
+	calories int `json:"calories"`
+}
+// setters
+func (f *Food) SetName(name string){
+	f.name = name
+}
+
+func (f *Food) SetCategory(c string){
```

Comment:
Should the variable be called c? You are also using c as an integer on l16. Perhaps just calling it category would be better (calories on l16)